guides/15_Background_Filter_Video_Processor.md

```diff
@@ -74,48 +74,73 @@ Cross-Origin-Embedder-Policy: require-corp
 
 when serving your application HTML. These headers are optional but advised: they improve web security, as well as allowing the browser to support technologies like `SharedArrayBuffer`.
 
-## Checking for support before offering background blur
+## Checking for support before offering background filtering
 
-Some browsers support the Amazon Chime SDK but do not support background blur. Additionally, some devices are not fast enough to keep up with real-time video while blurring the background.
+Some browsers support the Amazon Chime SDK but do not support background filtering. Additionally, some devices are not fast enough to keep up with real-time video while filtering the background.
 
 The SDK provides an async static method to allow you to cheaply check for the required browser features:
 
 ```typescript
+// Background Blur
 import { BackgroundBlurVideoFrameProcessor } from 'amazon-chime-sdk-js';
 await BackgroundBlurVideoFrameProcessor.isSupported();
+
+// Background Replacement
+import { BackgroundReplacemenetVideoFrameProcessor } from 'amazon-chime-sdk-js';
+await BackgroundReplacementVideoFrameProcessor.isSupported();
 ```
 
 If `isSupported` resolves to `true`, you can instantiate the blur video frame processor.
```

Comment:
>you can instantiate the blur video frame processor.

instantiate the background filter video frame process.

Reply:
fixed